.github/workflows/release-python-grpc-web.yml

```diff
@@ -0,0 +1,171 @@
+name: Release Python with grpc-web clients
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'grpc-web-rc'
+  workflow_dispatch:
+    description: "Manually publish release"
+    inputs:
+        publishWheels:
+          description: "Build and publish wheels qcs-sdk-python-grpc-web wheels to PyPI"
+          type: boolean
+          default: false
+
+jobs:
+  is-python-release:
+    if: true || github.event_name == 'workflow_dispatch' || startsWith(github.event.release.tag_name, 'python/v')
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Release tag starts with python/v, proceeding with release"
+  should-publish-wheels:
+    if: true || (github.event_name == 'workflow_dispatch' && inputs.publishWheels) || (github.event_name == 'release' && !github.event.release.prerelease)
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Publishing wheels"
+
+  macos:
+    runs-on: macos-12
+    needs: [is-python-release, should-publish-wheels]
+    env:
+      CXXFLAGS: "-std=c++11 -stdlib=libc++"
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Install protoc
+        uses: arduino/setup-protoc@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          version: '3.20.1'
+      - name: Patch package metadata for grpc-web
+        run: |
+          pip install toml
+          python crates/python/scripts/patch_package_name.py
+      - name: Build wheels - universal2
+        # universal2 supports both x86_64 and aarch64
+        uses: messense/maturin-action@v1
+        with:
+          args: -i python --release --target universal2-apple-darwin --manifest-path crates/python/Cargo.toml --features grpc-web --out dist
+      - name: Install wheel
+        run: |
+          pip install qcs-sdk-python --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels-grpc-web
+          path: dist
+
+  linux:
+    runs-on: ubuntu-latest
+    needs: [is-python-release, should-publish-wheels]
+    env:
+      CXXFLAGS: "-std=c++11"
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        target: [x86_64, aarch64]
```

Comment:
Will make an issue to follow-up with support for PowerPC. Builds for it currently fail due to an older version of `ring` that some of our dependencies use. The latest version of ring _does_ support PowerPC though, so we should be good if we can get our dependencies to all use ring >= 0.17

Reply:
Ultimately, we had to fork https://github.com/rigetti/hyper-proxy to get that dependency working. We patch it before building the python crate.